### ISSUE TITLE
Get jquery errors on Kibana 6.1.1

### ISSUE THREAD

**Issue by kawikao** [prelert/kibana-swimlane-vis#37]
```-> bin/kibana-plugin install https://github.com/prelert/kibana-swimlane-vis/releases/download/v6.1.1/prelert_swimlane_vis-6.1.1.zip
Attempting to transfer from https://github.com/prelert/kibana-swimlane-vis/releases/download/v6.1.1/prelert_swimlane_vis-6.1.1.zip
Transferring 13746 bytes....................
Transfer complete
Retrieving metadata from plugin archive
Extracting plugin archive
Extraction complete
Optimizing and caching browser bundles...
Plugin installation was unsuccessful due to error "Optimizations failure.
   3642 modules

    ERROR in ./plugins/prelert_swimlane_vis/public/prelert_swimlane_vis_controller.js
    Module not found: Error: Can't resolve 'flot-charts/jquery.flot' in '/usr/share/kibana/plugins/prelert_swimlane_vis/public'

    ERROR in ./plugins/prelert_swimlane_vis/public/prelert_swimlane_vis_controller.js
    Module not found: Error: Can't resolve 'flot-charts/jquery.flot.selection' in '/usr/share/kibana/plugins/prelert_swimlane_vis/public'

    ERROR in ./plugins/prelert_swimlane_vis/public/prelert_swimlane_vis_controller.js
    Module not found: Error: Can't resolve 'flot-charts/jquery.flot.time' in '/usr/share/kibana/plugins/prelert_swimlane_vis/public'```

**Comment by peteharverson** [comment]
Many apologies @kawikao . The `import` for the flot chart library needed updating as the location of that library changed in Kibana 6.1.0. I have pushed up a fix and updated the v6.1.1 build on the releases page.

You should be able to remove your broken install with `bin/kibana-plugin remove prelert_swimlane_vis` or just by deleting the `prelert_swimlane_vis` folder under the kibana `plugins` directory, and then re-run 
`bin/kibana-plugin install https://github.com/prelert/kibana-swimlane-vis/releases/download/v6.1.1/prelert_swimlane_vis-6.1.1.zip` to install the updated release.

I will also update the v6.1.0 release with the fix.

Please update this issue if you run into any problems.

**Comment by kawikao** [comment]
Worked like a charm. Cheers!
